src/test/scala/tech/sourced/swivel/small/DictionarySmallTest.scala

```diff
@@ -72,4 +72,18 @@ class DictionarySmallTest extends FunSuite {
     assert(vocab("b") == 1)
   }
 
+  test("read vocabulary: local path") {
+    // given
+    val path = "src/test/resources/row_vocab.txt"
+
+    // when
```

Comment:
For future code: if you like this style, you can use scalatest `GivenWhenThen`. See:
http://www.scalatest.org/user_guide/selecting_a_style